src/funtracks/import_export/import_from_geff.py

```diff
@@ -175,8 +171,13 @@ def import_from_geff(
         Tracks based on the geff graph and segmentation, if provided.
     """
 
-    group = zarr.open_group(directory, mode="r")
-    metadata = dict(group.attrs)
+    # Read the GEFF file into memory
+    in_memory_geff = read_to_memory(directory)
```

Comment:
Pass in an argument for node_props and edge_props to limit reading the properties we need